### DISCUSSION TITLE
fedml.init()

### DISCUSSION THREAD

**Discussion by rekkles2** [FedML-AI/FedML#1521] [Q&A]
### When I run `fedml.init()`，I get an error:
raise Exception(
Exception: no such a platform. training_type = None, backend = None
Traceback (most recent call last):

- line 8, in <module>

    args = fedml.init()

- line 66, in init

    args = load_arguments(fedml._global_training_type, fedml._global_comm_backend)

- line 202, in load_arguments

    args = Arguments(cmd_args, training_type, comm_backend)

- line 84, in __init__

    self.get_default_yaml_config(cmd_args, training_type, comm_backend)

- line 133, in get_default_yaml_config

    raise Exception(
**_There is no documentation telling me what they mean and I don't know how to solve this problem_**

**Comment by Galaxyphan** [comment]
Pls did you solve this problem

**Comment by Galaxyphan** [comment]
You can't run it directly, but you should give it a configuration file，u can view run_client.sh that file

**Reply by rekkles2** [reply]
thanks
